AnimView/frmMain.cpp

```diff
@@ -66,25 +67,40 @@ BEGIN_EVENT_TABLE(frmMain, wxFrame)
   EVT_CHECKBOX(ID_DRAW_COORDINATES, frmMain::_onToggleDrawCoordinates)
 END_EVENT_TABLE()
 
+namespace {
+constexpr int make_id(int layer, int id) {
+  return frmMain::ID_LAYER_CHECKS + (layer * 25) + id;
+}
+}  // namespace
+
 frmMain::frmMain()
     : wxFrame(nullptr, wxID_ANY, L"Theme Hospital Animation Viewer") {
   wxSizer* pMainSizer = new wxBoxSizer(wxHORIZONTAL);
 
   wxSizer* pSidebarSizer = new wxBoxSizer(wxVERTICAL);
 
-#define def wxDefaultPosition, wxDefaultSize
+  wxSizerFlags fillSizerFlags(0);
+  fillSizerFlags.Expand().Border(wxALL, 0);
+
+  wxSizerFlags fillBorderSizerFlags(0);
+  fillSizerFlags.Expand().Border(wxALL, 1);
```

Comment:
Missing "Border" in the variable name?

Reply:
Alright. I bet that changed appearances. Didn't notice from a sight inspection.